src/tests/work-item/work-item-list/assigne.spec.js

```diff
@@ -0,0 +1,149 @@
+ *  https://github.com/almighty/almighty-core/issues/298
+ *
+ * Note on screen resolutions - See: http://www.itunesextractor.com/iphone-ipad-resolution.html
+ * Tests will be run on these resolutions:
+ * - iPhone6s - 375x667
+ * - iPad air - 768x1024
+ * - Desktop -  1920x1080
+ *
+ * beforeEach will set the mode to phone. Any tests requiring a different resolution will must set explicitly.
+ *
+ * @author ldimaggi
+ */
+
+var WorkItemListPage = require('./work-item-list.page'),
+  testSupport = require('./testSupport');
+
+describe('Work item list', function () {
+  var page, items, startCount, browserMode;
+
+var until = protractor.ExpectedConditions;
+var waitTime = 30000;
+
+  beforeEach(function () {
+    testSupport.setBrowserMode('phone');
+    page = new WorkItemListPage(true);
+    page.allWorkItems.count().then(function(originalCount) { startCount = originalCount; });
+  });
+/**Test searching user in the assignee drop down  */
+  it('Test searching user in the assignee drop down -phone ', function() {
+      var workItemTitle = "The test workitem title";
+      var workItemUpdatedTitle = "The test workitem title - UPDATED";
+      page.clickWorkItemQuickAdd();
+      page.typeQuickAddWorkItemTitle(workItemTitle);
+      page.clickQuickAddSave().then(function() {
+      page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
+      detailPage.setWorkItemDetailAssigneeSearch("Harry Potter",false);
+      detailPage.clickAssignedUserDropDownList("Harry Potter");
+      expect(detailPage.details_assigned_user().getText()
+      ).toContain("Harry Potter");
+      detailPage.clickWorkItemDetailCloseButton();
+      });
+    });
+  }); 
+  /**Test able to click assigne button Icon and Assignee Name   */
+  it('Test able to click assigne button Icon and Assignee Name -phone ', function() {
+      var workItemTitle = "The test workitem title";
+      var workItemUpdatedTitle = "The test workitem title - UPDATED";
+      page.clickWorkItemQuickAdd();
+      page.typeQuickAddWorkItemTitle(workItemTitle);
+      page.clickQuickAddSave().then(function() {
+      page.workItemViewId(page.workItemByTitle(workItemTitle)).getText().then(function (text) {
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      expect(detailPage.workItemDetailAssigneeIcon().isPresent()).toBeTruthy();
+      expect(detailPage.workItemDetailAssignee().isPresent()).toBeTruthy();
+      detailPage.clickWorkItemDetailCloseButton();
+      });
+    });
+  }); 
+ 
+  /**Test to update the assigned user */ 
+  it('Test to update the assigned user  -phone ', function() {
+      page.workItemViewId(page.firstWorkItem).getText().then(function (text) { 
+      var detailPage = page.clickWorkItemTitle(page.firstWorkItem, text);
+      detailPage.workItemDetailAssigneeIcon().click();
```

Comment:
For safety's sake, I suggest first verifying that the assigned user is not 'Walter Mitty' before setting the assigned user to be 'Walter Mitty.'  Just in case...   ;-)